beacon/goclient/goclient.go

```diff
@@ -65,7 +65,7 @@ var (
 func init() {
 	for _, c := range allMetrics {
 		if err := prometheus.Register(c); err != nil {
-			log.Println("could not register prometheus collector")
```

Comment:
revert this change it is incorrect. the logger is not setup in init functions yet